src/Parsing.hs

```diff
@@ -4,6 +4,7 @@ module Parsing (parse, validCharacters, balance) where
 
 import Text.Parsec ((<|>))
 import qualified Text.Parsec as Parsec
+import qualified Data.Char as Char
```

Comment:
I just realized I introduced this spurious import here, @eriksvedang! Sorry about that!

Reply:
Its not needed?

Reply:
No, I added it when trying to patch the parser for non-capitalized names (did not work out, though, for obvious reasons).

Reply:
OK, I’ll remove it then, cheers.